amgut/db/patches/0010.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE ag.external_survey ( 
+	external_survey_id   bigserial  NOT NULL,
+	external_survey      varchar  NOT NULL,
+	external_survey_description varchar  ,
```

Comment:
I think I misunderstood the purpose of this table. If this is just storing the name of the survey and giving it a handle in our database, then I think @wasade is correct and it should be renamed to `external_survey_types` (or something like `external_survey_sources` if you don't like `types`)